src/entity/notifications/Notification.ts

```diff
@@ -71,6 +71,9 @@ export class Notification {
   @Column({ type: 'text', default: '0' })
   uniqueKey?: string;
 
+  @Column({ type: 'int', nullable: true })
+  numTotalAvatars?: number;
```

Comment:
Nice! 
Perhaps something we can note for the [notification DR](https://dailydotdev.atlassian.net/jira/software/projects/WT/boards/11?selectedIssue=WT-1936) to use in other places.

Reply:
Added a link to this comment in the jira issue for the notification DR